src/content/manager/analysis.manager.ts

```diff
@@ -2,7 +2,7 @@ import { BaseManager } from "./base.manager";
 import { ManagerConfig } from "../../interfaces/manager-config.interface";
 import { AssetManager } from "./asset.manager";
 import * as fs from "fs";
-import {stringify} from "../../util/yaml";
+import {stringify} from "../../util/json";
```

Comment:
These are STUDIO related operations for which we have not made any changes. Can we limit the scope of this PR to core endpoints only? Everything else should work the same as it did before.

Reply:
Reverted them back in [this commit](https://github.com/celonis/content-cli/pull/186/commits/deaa7a423ecf70e636bd91f212a734f7aa4d6b94).